lib/model/model.go

```diff
@@ -2376,6 +2374,136 @@ func (m *Model) GlobalDirectoryTree(folder, prefix string, levels int, dirsonly
 	return output
 }
 
+func (m *Model) GetFolderVersions(folder string) (map[string][]interface{}, error) {
+	fcfg, ok := m.cfg.Folder(folder)
+	if !ok {
+		return nil, errFolderMissing
+	}
+
+	loc, err := time.LoadLocation("Local")
+	if err != nil {
+		return nil, err
+	}
+
+	files := make(map[string][]interface{})
```

Comment:
Why not `map[string][]FileVersion` ?

Reply:
Gui does a good job not to depend on lib/model, this would have to changed if we want to return something from the model package there.

Reply:
Ugh, yeah. Hmm. Would it be super ugly to make this struct a `versioner.Version` or something? Otherwise at least a comment on what's going on here would be nice...